ports/bzip2/portfile.cmake

```
@@ -2,6 +2,7 @@ vcpkg_from_git(
     OUT_SOURCE_PATH SOURCE_PATH
     URL https://sourceware.org/git/bzip2.git
     REF 75a94bea3918e612b879d6a11ca64b8689526147
```

Could you please add the following comment showing the version after REF:
```suggestion
    REF 75a94bea3918e612b879d6a11ca64b8689526147 # 1.0.8
```

---

CW